internal/client/telegram.go

```diff
@@ -0,0 +1,105 @@
+package client
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/base-org/pessimism/internal/core"
+	"github.com/base-org/pessimism/internal/logging"
+)
+
+type TelegramClient interface {
+	AlertClient
+}
+
+type TelegramConfig struct {
+	Token  string
+	ChatID string
```

Comment:
Can we add some docstring links to telegram documentation that gives better introspection as to what these values mean?